src/mesh/data/gridfromfile.cxx

```diff
@@ -84,7 +84,13 @@ bool GridFile::get(Mesh *UNUSED(m), int &ival,      const string &name) {
   if(!file->is_valid())
     throw BoutException("File cannot be read");
   
-  return file->read(&ival, name);
+  bool success = file->read(&ival, name);
+  if(success){
+	  output << "\tOption " << name  << " = " << ival << " (" << filename <<")" << endl;
+  }
```

Comment:
Arghh stupid tabs in Eclipse :-D. I was so produ becuase it looked perfect in my editor ;-)